crates/bevy_gizmos/src/lib.rs

```diff
@@ -194,20 +194,14 @@ impl AppGizmoBuilder for App {
         self.init_resource::<GizmoStorage<T>>()
             .add_systems(Last, update_gizmo_meshes::<T>);
 
-        let Ok(render_app) = self.get_sub_app_mut(RenderApp) else {
-            return self;
-        };
-
-        render_app.add_systems(ExtractSchedule, extract_gizmo_data::<T>);
-
         self
     }
 }
 
 #[derive(Resource, Default)]
 struct LineGizmoHandles {
-    list: TypeIdMap<Handle<LineGizmo>>,
-    strip: TypeIdMap<Handle<LineGizmo>>,
+    list: TypeIdMap<Option<Handle<LineGizmo>>>,
```

Comment:
A comment on why this is an `Option` would be helpful. I can figure it out based on the PR description, but without context a map storing an option just looks like a code quality issue.